shared/lib/deep-links/routes/index.ts

```diff
@@ -33,6 +33,11 @@ export function addRoute(route: Route) {
   routes.set(route.pathname, route);
 }
 
+if (process.env.ENABLE_SETTINGS_PAGE_DEV_OPTIONS || process.env.IN_TEST) {
+  // eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires, node/global-require
+  addRoute(require('./test-route').default);
+}
+
```

Comment:
All changes in this file come from this PR https://github.com/MetaMask/metamask-extension/pull/38142, which was not included in 13.11.0, but was cherry-picked in 13.10.4.